packages/cozy-harvest-lib/src/konnector-policies.js

```diff
@@ -1,6 +1,7 @@
 import logger from './logger'
 import { konnectorPolicy as biKonnectorPolicy } from './services/budget-insight'
 import { konnectorPolicy as biWebViewPolicy } from './services/biWebView'
+import { konnectorPolicy as cccPolicy } from './services/cccPolicy'
```

Comment:
ccc means connecteur côté client, this is not the word we used. Name it clisk 🙏 . 

Also, other policies are namde `budget-insight` or `biWebview`. There is no `Policy` within the name.

Reply:
Ok for clisk.

For the second part, I would prefer then to move the policies in their own directory named policies to make it clearer